app/src/main/java/fr/free/nrw/commons/contributions/ContributionsListFragment.java

```diff
@@ -99,14 +105,21 @@ public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
     }
 
     private void initRecyclerView() {
+        Timber.d("RecyclerList Recycler view Init.");
+        LinearLayoutManager layoutManager;
         if (getResources().getConfiguration().orientation == Configuration.ORIENTATION_LANDSCAPE) {
-            rvContributionsList.setLayoutManager(new GridLayoutManager(getContext(),SPAN_COUNT));
+            layoutManager = new GridLayoutManager(getContext(), SPAN_COUNT);
```

Comment:
a GridLayoutManager with spanCount 1 is essentially analogous to LinearLayoutManager, I would believe

That way we could move this condition into a ternary and not create different types of LayoutManager

Reply:
We are using span count = 3.

Reply:
Yes we are but Instead of a LinearLayoutManager and a GridLayoutManager we could use only a GridLayoutManager and based on Orientation set the span count